cmd/queue/main.go

```diff
@@ -198,6 +205,10 @@ func probeUserContainer() bool {
 // Make handler a closure for testing.
 func handler(reqChan chan queue.ReqEvent, breaker *queue.Breaker, handler http.Handler) func(http.ResponseWriter, *http.Request) {
 	return func(w http.ResponseWriter, r *http.Request) {
+		recorder := pkghttp.NewResponseRecorder(w, http.StatusOK)
+		reqCtx, fullQueueProxySpan := trace.StartSpan(r.Context(), "queueproxy")
```

Comment:
As we're overloading `reqCtx` a couple of times. Maybe this should be `fullCtx` and the others are `probeCtx` or `proxyCtx` respectively.

On the 3 span discussion: isn't the outer span automatically generated via the middleware?

Reply:
Looks like youre correct, ochttp should be making a span for the duration of the http handler - https://github.com/census-instrumentation/opencensus-go/blob/b7bf3cdb64150a8c8c53b769fdeb2ba581bd4d4b/plugin/ochttp/server.go#L108

We may not be setting this up properly though as @beemarie reports not seeing this span...

Reply:
The span is there actually - I was using a browser to test my URL & there's both a span with /favicon & a span with / for that case.  I think these should represent the outer span, so we should be able to remove the "queueproxy" span & keep "probe" & "proxy"